contents/docs/libraries/angular.md

```diff
@@ -65,7 +65,71 @@ PostHog only captures pageview events when a [page load](https://developer.mozil
 
 If we want to capture every route change, we must write code to capture pageviews that integrates with the router.
 
-To do this, import `posthog` into `app-routing.module.ts`, subscribe to router events and then capture `$pageview` events on `NavigationEnd` events:
+To start, we need to add `capture_pageviews: false` to the PostHog initialization to avoid double capturing the first pageview.
+
+```ts file=main.ts
+import { bootstrapApplication } from '@angular/platform-browser';
+import { appConfig } from './app/app.config';
+import { AppComponent } from './app/app.component';
+import posthog from 'posthog-js'
+
+posthog.init(
+  '<ph_project_api_key>',
+  {
+    api_host:'<ph_client_api_host>',
+    person_profiles: 'identified_only',
+    capture_pageviews: false
+  }
+)
+
+bootstrapApplication(AppComponent, appConfig)
+  .catch((err) => console.error(err));
+```
+
+Next, to track pageviews in Angular, we import the following in `app.component.ts`:
```

Comment:
It's not super clear which version of Angular we're referring to here, and we shouldnt assume most people will be on the latest version. Adding subheadings to make it clearer:

```suggestion
Next, depending on your Angular version, add the following code to track pageviews:

### Anguglar v18 and above

Import the following in `app.component.ts`:
```